src/nft/components/profile/view/ViewMyNftsAsset.tsx

```diff
@@ -37,7 +37,7 @@ const getNftDisplayComponent = (
 const getUnsupportedNftTextComponent = (asset: WalletAsset) => (
   <Box as="span" className={bodySmall} style={{ color: themeVars.colors.textPrimary }}>
     {asset.asset_contract.tokenType === TokenType.ERC1155 ? (
-      <Trans>ERC-1155 support coming soon</Trans>
+      <Trans>Selling ERC-1155s is coming soon</Trans>
```

Comment:
we may need to either increase the width of the tooltip or maybe remove the `is` as the text is wrapping and doesn't look great, can test impersonating my wallet